.github/docker-images/ubuntu-16-x64/Dockerfile

```diff
@@ -24,14 +25,17 @@ RUN apt-get update -qq \
 ###############################################################################
 # Python/AWS CLI
 ###############################################################################
-RUN python3 -m pip install setuptools \
-    && python3 -m pip install --upgrade awscli \
-    && aws --version
+RUN python3 -m pip install setuptools
```

Comment:
Put these all in one command, otherwise docker will create a layer for each of these. That's why there's so many continuations in Dockerfiles